packages/redux/src/raffles/actions/factories/modifyRaffleParticipationFactory.ts

```diff
@@ -0,0 +1,54 @@
+import * as actionTypes from '../../actionTypes';
+import {
+  Config,
+  PatchRaffleParticipation,
+  PatchRaffleParticipationData,
+  Raffle,
+  toBlackoutError,
+} from '@farfetch/blackout-client';
+import type { Dispatch } from 'redux';
+
+/**
+ * Method responsible to create a raffle participation.
+ *
+ * @param postRaffleParticipation - Create raffle participation.
+ *
+ * @returns Thunk factory.
+ */
+const modifyRaffleParticipationFactory =
+  (patchRaffleParticipation: PatchRaffleParticipation) =>
+  (
+    raffleId: Raffle['id'],
+    participationId: string,
+    data: PatchRaffleParticipationData,
+    config?: Config,
+  ) =>
+  async (dispatch: Dispatch): Promise<number> => {
+    try {
+      dispatch({
+        type: actionTypes.MODIFY_RAFFLE_PARTICIPATION_REQUEST,
+      });
+      const result = await patchRaffleParticipation(
+        raffleId,
+        participationId,
+        data,
+        config,
+      );
+      // const normalizedResult = normalize(data, raffleParticipationSchema);
```

Comment:
I think this was correct

Reply:
I get a service status, so I just return the value (status changed) from the data object...
what do you suggest to do here?

Reply:
Didn't notice, it's all good then 😄